### DISCUSSION TITLE
What IOS app that works the best for ND?

### DISCUSSION THREAD

**Discussion by gerroon** [navidrome/navidrome#2627] [Q&A]
Hi

I am looking for the best experience on IOS> I tested free ones, not so happy. Any tested recommendations?Paid is fine.

**Comment by PassiveLemon** [comment]
I recommend Amperfy. It supports offline downloads and I have had a very great experience with it

**Reply by gerroon** [reply]
Really ? I get errors like this with it in my network, definetely not great experience for me 
![image](https://github.com/navidrome/navidrome/assets/8519469/0f7ec040-b018-4c31-bf65-a74d1b23da9f)

**Comment by dweymouth** [comment]
I haven't used it, but I've often seen play:Sub (paid) recommended as the best iOS app. If gapless playback is in your required feature set, your choices are just play:Sub or iSub (FOSS, old and unmaintained; works, but clunky UI)

**Reply by gerroon** [reply]
thanks I will take a look at it

**Comment by tty228** [comment]
The application I am most satisfied with is this one, but it should not support multiple languages for the time being: https://www.appinn.com/yinliu-music-player/

github: https://github.com/gitbobobo/StreamMusic

**Reply by gerroon** [reply]
Thanks, what is it called in the IOS app store, I cant read the language and streammusic is not listed in the app store.

**Reply by tty228** [reply]
https://apps.apple.com/us/app/stream-music-enjoy-music/id6449966496

**Comment by ghost** [comment]
I recommend play:Sub or iSub.

**Reply by gerroon** [reply]
Thanks I will take a look at them for real. Too bad that ISub's privacy policy does not look good.

**Reply by tirtawn** [reply]
I am using avsub. You will like the data privacy policy.

**Comment by ghost** [comment]
> Too bad that ISub's privacy policy does not look good.

I'm not sure what that means. iSub has one great advantage: it is now free and open source. In my fork I have fixed numerous bugs and am currently modernizing the code.

**Reply by gerroon** [reply]
Thanks for the reply. The version on the app store says the latest release was like 2 years. ago. Are you planning to do a new release soon?

**Reply by ghost** [reply]
Oh, I don't know about that. My point is merely that a project that has been open-sourced and that others can successfully compile and run is very much alive.

**Comment by mattneub** [comment]
In the end I wrote my own: https://github.com/navidrome/navidrome/discussions/3876

**Comment by Metal-Snake** [comment]
So far I tried these 6 apps, each of them has their pros and cons. 

flo https://github.com/kepelet/flo (it requires https when using outside the local network)

substreamer https://substreamerapp.com/

amperfy https://github.com/BLeeEZ/amperfy

**stream music** https://github.com/gitbobobo/StreamMusic/blob/main/README_en.md - https://apps.apple.com/us/app/stream-music-enjoy-music/id6449966496 - This one is my favourite currently. 

Arpeggi https://www.reddit.com/r/arpeggiApp/ (currently only as a Testflight version, but works really good so far)

AginMusicMobile https://github.com/TymekV/AginMusicMobile - still a very early version, looks promising
